lib/guessingGameMakeGuess.js

```diff
@@ -2,7 +2,16 @@
  * Given a game and guess,
  */
 function guessingGameMakeGuess(game, guess) {
-
+  game.numGuesses++;
+  if (game.targetWord.includes(guess)) {
```

Comment:
`game.targetWord.includes` will have to iterate over `targetWord` to do its work, so no real reason to check before iterating.